manifests/addons/dashboards/istio-service-dashboard.json

```diff
@@ -2984,7 +2984,7 @@
     ]
   },
   "timezone": "",
-  "title": "Istio Service Dashboard",
+  "title": "Istio K8s Service Dashboard (Deprecated)",
```

Comment:
This is not necessarily only about k8s. The service can be hosted anywhere.

Reply:
Any suggestions for a better name?  Putting kubernetes isn't perfect, but I'm struggling to think of a better concept here. There is obvious overlap at the edges with the new dashboard.